scripts/all-helpers.sh

```diff
@@ -68,10 +68,9 @@ helper_libtestdriver1_adjust_config() {
     fi
 
     # Enable PSA-based config (necessary to use drivers)
-    # The configuration option has been removed for 4.0. While the project and
-    # branch detection shell in shell work is not completed, just ignore the
-    # failure to set MBEDTLS_PSA_CRYPTO_CONFIG.
-    scripts/config.py set MBEDTLS_PSA_CRYPTO_CONFIG || true
```

Comment:
I would keep some comment for future context. People modifying this code need to know this is a 3.6 specific setting. 

e.g 

> `MBEDTLS_PSA_CRYPTO_CONFIG` is a legacy setting which should only be set on 3.6 LTS branches.